values.yaml

```diff
@@ -280,7 +280,7 @@ persistence:
     # "oss" and fill the information needed in the corresponding section. The type
     # must be "filesystem" if you want to use persistent volumes for registry
     # and chartmuseum
-    type: filesystem
+    type: gcs
```

Comment:
please keep the default `type` as "filesystem", do not change it to gcs

Reply:
Hi,
of course, fixed !